store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -532,106 +567,161 @@ private boolean validateBoolean(String param, String value) throws ParseExceptio
+        options.addOption(new Option(O_SMC, A_STORE_MANAGER_CLASS, true, H_DESC_VOL_SMC));
+
+        // OpenIO
+        options.addOption(new Option(O_AP, A_ACCOUNT_PORT, true, H_DESC_VOL_AP));
+        options.addOption(new Option(O_PP, A_PROXY_PORT, true, H_DESC_VOL_PP));
+        options.addOption(new Option(O_URL, AdminConstants.A_VOLUME_URL, true, H_DESC_VOL_URL));
+        options.addOption(new Option(O_NS, A_NAME_SPACE, true, H_DESC_VOL_NS));
+        options.addOption(new Option(O_ACC, AdminConstants.A_ACCOUNT, true, H_DESC_VOL_ACC));
     }
 
     @Override
     protected void usage(ParseException e) {
         if (e != null) {
-            System.err.println("Error parsing command line arguments: " + e.getMessage());
+            System.err.println(STR_ERR_PARSE_CLI_ARG + SPACE + e.getMessage());
         }
-
-        System.err.println(getCommandUsage());
-        printOpt(O_A, 0);
-        printOpt(O_N, 2);
-        System.err.println(HELP_EXTERNAL_NAME);
-        printOpt(O_T, 2);
-        printOpt(O_P, 2);
-        printOpt(O_C, 2);
-        printOpt(O_CT, 2);
-        printOpt(O_E, 0);
-        printOpt(O_ID, 2);
-        System.err.println("  any of the options listed under -a can also be specified " );
-        System.err.println("  to have its value modified.");
-        printOpt(O_D, 0);
-        printOpt(O_ID, 2);
-        printOpt(O_L, 0);
-        printOpt(O_ID, 2);
-        System.err.println("  -id is optional.");
-        printOpt(O_DC, 0);
-        printOpt(O_SC, 0);
-        printOpt(O_ID, 2);
-        printOpt(O_TS, 0);
-        printOpt(SoapCLI.O_AUTHTOKEN, 0);
-        printOpt(SoapCLI.O_AUTHTOKENFILE, 0);
-        printOpt(O_ST, 0);
-        printOpt(O_VP, 0);
-        printOpt(O_STP, 0);
-        printOpt(O_BID, 0);
-        printOpt(O_UIT, 0);
-        printOpt(O_UFA, 0);
-        printOpt(O_UFAT, 0);
-
-        printOpt(O_PP, 0);
-        printOpt(O_AP, 0);
-        printOpt(O_ACCOUNT, 0);
-        printOpt(O_URL, 0);
-        printOpt(O_NS, 0);
-
-        printOpt(O_SMC, 0);
+        printLineWithLeftPad(getCommandUsage(), 0);
+
+        printLineWithLeftPad(STR_EMPTY, 0);
+        printLineWithLeftPad(H_TLE_INT_VOL_ADD, 0);
+        printOptWithDesc(O_A,   4, H_DESC_VOL_ADD,        null);
+        printOptWithDesc(O_N,   8, H_DESC_VOL_NAME,       true);
+        printOptWithDesc(O_T,   8, H_DESC_VOL_TYPE_PSI,   true);
+        printOptWithDesc(O_P,   8, H_DESC_VOL_RP,         true);
+        printOptWithDesc(O_ST,  8, H_DESC_VOL_STI,        false);
+        printOptWithDesc(O_C,   8, H_DESC_VOL_CB,         false);
+        printOptWithDesc(O_CT,  8, H_DESC_VOL_CT,         false);
+        printOptWithDesc(O_SMC, 8, H_DESC_VOL_SMC,        false);
+        printLineWithLeftPad(H_EXP_INT_VOL_ADD, 4);
+
+        printLineWithLeftPad(STR_EMPTY, 0);
+        printLineWithLeftPad(H_TLE_EXT_S3_VOL_ADD, 0);
+        printOptWithDesc(O_A,     4, H_DESC_VOL_ADD,        null);
```

Comment:
We can print only once, we don't print it again and again

Reply:
Can you explain this point ?

Reply:
You are calling printOptWithDesc method for every line to print so instead of printing every line one by one you can use StringBuilder for the output and print once when all the output is stored in StringBuilder. 

Inside the method printOptWithDesc we are creating the object of StringBuilder for the call so instead of creating the new object, you can use a single object here.

Reply:
I think we shall keep logic of printing cli option one by one as it will help us to maintain it better identify anf fix bugs. 
This is approach is also error free and free from space and alignment issues.